manual/googlechrome-extensions/googlechrome-emberinspector/googlechrome-ember-inspector.nuspec

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-ember-inspector</id>
+    <version>1.0.0</version>
+    <title>Ember Inspector</title>
+    <authors>Ember Development Team</authors>
+    <owners>gep13, chocolatey</owners>
```

Comment:
Small one: chocolatey should be listed first according to [§1.2.4](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#124-add-chocolatey-among-owners)